app/src/components/Sections/Footer.tsx

```diff
@@ -28,10 +28,13 @@ const Footer = ({ lng }: { lng: string }) => {
           >
             <FooterLink url="/" title={t("about-citycatalyst")} />
             <FooterLink url="/" title={t("contribution-guide")} />
-            <FooterLink url="/" title={t("goto-gh")} />
+            <FooterLink
+              url="https://github.com/Open-Earth-Foundation/CityCatalyst"
+              title={t("goto-gh")}
+            />
             <FooterLink url="/" title={t("cad")} />
             <FooterLink url="/" title={t("read-the-docs")} />
-            <FooterLink url="/" title={t("python-client-docs")} />
+            <FooterLink url="./cdp" title={t("submit-to-cdp")} />
```

Comment:
This is a relative path, so it will append /cdp to the current page URL. This is also why I enabled trailing slashes in the NextJS config.

I didn't want to have to pass the current inventory ID into the footer component, but that might be a more direct way to handle this and not rely on the paths of the pages.